src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -132,50 +109,91 @@ TEST(PipeLoggerTestWithTee, RedirectionWithTee) {
   // Check log content after completion.
   EXPECT_EQ(CompleteReadFile(test_file_path),
             absl::StrFormat("%s%s", kLogLine1, kLogLine2));
-
-  // Delete temporary file.
-  EXPECT_EQ(unlink(test_file_path.data()), 0);
 }
 
 TEST(PipeLoggerTestWithTee, RotatedRedirectionWithTee) {
```

Comment:
Actually, shouldn't we do `TEST_P(PipeLoggerTest, RotatedRedirectionWithTee) {` so that we can also test different pipe sizes?

Reply:
In the test file, we have a few things to test: (internal impl) pipe vs no pipe, (internal impl) pipe size's effect, features.

On pipe size, we already test it above, I don't understand why do we need to add more tests here?